client/me/profile-links/index.jsx

```diff
@@ -175,33 +160,39 @@ module.exports = React.createClass( {
 					? this.renderProfileLinks()
 					: this.renderNoProfileLinks()
 				}
+			</div>
+		);
+	},
 
-				{
-					'wordpress' === this.state.showingForm
-					? (
-						<ProfileLinksAddWordPress
-							userProfileLinks={ this.props.userProfileLinks }
-							onSuccess={ this.hideForms }
-							onCancel={ this.hideForms }
-						/>
-					)
-					: null
-				}
-
-				{
-					'other' === this.state.showingForm
-					? (
-						<ProfileLinksAddOther
-							userProfileLinks={ this.props.userProfileLinks }
-							onSuccess={ this.hideForms }
-							onCancel={ this.hideForms }
-						/>
-					)
-					: null
-				}
-
-				{ ! this.state.showingForm ? this.renderFormVisibilityControls() : null }
-		</div>
+	render: function() {
+		return(
+			<div>
+				<SectionHeader label={ this.translate( 'Profile Links' ) }>
```

Comment:
:+1: I'll see if we can revisit this before more sections start using it this way.